serverless.yaml

```diff
@@ -17,6 +17,7 @@ custom:
   apigatewayClientErrorThreshold: ${env:apigatewayClientErrorThreshold}
   lambdaErrorThreshold: ${env:lambdaErrorThreshold}
   ddbToESLambdaErrorThreshold: ${env:ddbToESLambdaErrorThreshold}
+  archiveConfig: ${env:archiveConfig, ''}
```

Comment:
I know there are other existing env vars set here, but should the naming convention for env var names be ALL_CAPS? Also, should be push the default into here?